proto/serverless/instrumentation/tags/v1/error.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package serverless.instrumentation.tags.v1;
+
+option go_package = ".;protoc";
+
+message ErrorTags {
+    // The Error Name
+    string name = 1;
+    // The Error Message - Depending on runtime this is defined or not.
+    optional string message = 2;
+    // The Error stacktrace if applicable
+    optional string stacktrace = 3;
+
+    enum ErrorType {
+        // No ErrorType was provided. This should never be the case and if it is received ingest will ignore it.
+        ERROR_TYPE_UNSPECIFIED = 0;
+
+        // An Uncaught exception was raised that interupts execution unexpectedly. 
+        ERROR_TYPE_UNCAUGHT = 1;
+       
+        // An Uncaught exception, specifically a timeout of the compute environment,
+        // A timeout in Lambda, for example, is registered when ingest receives a log line
+        // stating that the task timed out.
+        ERROR_TYPE_UNCAUGHT_TIMEOUT = 2;
+
+        // An Uncaught exception, specifically the compute environment ran out of memory.
+        ERROR_TYPE_UNCAUGHT_OOM = 3;
+
+        // An uncaught exception that takes place during the AWS Lambda Initialization phase.
```

Comment:
@Mmarzex Does it cover both (1) handler not found and (2) handler crashed at initialization?

Reply:
@medikoo Is that the intention we have for it? I'm fine with it covering both.

Reply:
Originaly intention was to handle just _handler not found_ errors here, and eventually split one responsible for uncaught exceptions or unhandled rejections to one that covers invocation and other that covers initialization

Reply:
> Originaly intention was to handle just handler not found errors here, and eventually split one responsible for uncaught exceptions or unhandled rejections to one that covers invocation and other that covers initialization

Then propose those types please. We talked about this in the meeting on monday and these were the types we all agreed upon.